suppressionFile.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<suppressions xmlns="https://jeremylong.github.io/DependencyCheck/dependency-suppression.1.3.xsd">
```

Comment:
pipeline should show a warning (if possible) if a vulnerability is found. if we ignore them here - we will forget that they exist.
Let's remove the suppression file.

Reply:
Removed the suppression file- 
Seams like something like allow-failure is not implemented yet https://github.com/actions/toolkit/issues/399

Reply:
yes. I readded the supression as we can only fail or pass a step :/